roles/ovirt-host-deploy/meta/main.yml

```diff
@@ -0,0 +1,22 @@
+galaxy_info:
+  author: Ondra Machacek
+  description: Role to deplo oVirt hypervisor.
+  company: Red Hat, Inc.
+
+  license: Apache License 2.0
+
+  min_ansible_version: 2.3
+
+  platforms:
+  - name: EL
+    versions:
+    - 7
+  - name: Fedora
+    versions:
+    - 24
```

Comment:
I think we can remove FC24, oVirt 4.2 will most probably support only FC26 (but we haven't started FC26 builds yet)